lib/ZwaveClient.js

```diff
@@ -1838,6 +1838,12 @@ ZwaveClient.prototype.getInfo = function () {
   return info
 }
 
+ZwaveClient.prototype.refreshValues = function (nodeId) {
+  this.getNode(nodeId).refreshValues()
```

Comment:
Anyway I prefer something like: 

```js
ZwaveClient.prototype.refreshValues = async function (nodeId) {
  if (this.driver && !this.closed) {
    const zwaveNode = this.getNode(nodeId)

    return zwaveNode.refreshValues()
  }

  throw Error('Driver is closed')
}
```

Reply:
Should it be a throw or a catch per @AlCalzone comment?

Reply:
also should it still return true?

Reply:
This is how I have implemented all other methods if you check. It's correct to throw there